Description: Making the server packages ship dynamic shared libraries aswell.
--- a/sched/Makefile.am
+++ b/sched/Makefile.am
@@ -4,7 +4,7 @@
 include $(top_srcdir)/Makefile.incl
 
 AM_CPPFLAGS += $(MYSQL_CFLAGS) $(PTHREAD_CFLAGS)
-AM_LDFLAGS += -static -L../lib/.libs/
+AM_LDFLAGS += -L../lib/.libs/
 
 if ENABLE_LIBRARIES
 
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -44,7 +44,7 @@
     gui_urls.xml
 
 AM_CXXFLAGS += $(MYSQL_CFLAGS)
-AM_LDFLAGS += -static -L../lib/.libs -L../sched
+AM_LDFLAGS += -L../lib/.libs -L../sched
 
 cancel_jobs_SOURCES = cancel_jobs.cpp
 cancel_jobs_LDADD = $(SERVERLIBS)
--- a/vda/Makefile.am
+++ b/vda/Makefile.am
@@ -4,7 +4,7 @@
 vda_PROGRAMS = vda vdad ssim
 
 AM_CXXFLAGS += $(MYSQL_CFLAGS)
-AM_LDFLAGS += -static -L../lib/.libs -L../sched
+AM_LDFLAGS += -L../lib/.libs -L../sched
 
 vda_SOURCES = vda.cpp vda_lib.cpp vda_lib2.cpp vda_policy.cpp stats.cpp
 vda_LDADD = $(SERVERLIBS)
